src/EditableDialogBox/LayoutItem.php

```diff
@@ -27,6 +27,9 @@ protected function addItem(DialogBoxItem $item): static
         return $this;
     }
 
+    /**
+     * @return array<int|string, mixed>
+     */
```

Comment:
I don't think we need this, it should be inherited. Did PhpStan complain?

Reply:
Exactly that happens...